src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -44,7 +44,7 @@ namespace Tizen.NUI.Components
     /// </code>
```

Comment:
The sentence in line 31 needs a fix. It could be simplified to "If current top Page and next top Page have same \<see cref="View.TransitionOptions.TransitionTag"\\\>, Navigator creates smooth transition motion for them." or at least "those have" -> "which have"

Reply:
Thank you, you are right.